Podfile

```diff
@@ -12,6 +12,7 @@ target 'Signal' do
     pod 'Reachability'
     pod 'SignalServiceKit',           path: '.'
     pod 'SocketRocket',               :git => 'https://github.com/facebook/SocketRocket.git'
+	pod 'YYImage'
```

Comment:
In my experience YYImage is far more efficient and reliable than FLAnimatedImage for animating GIFs, etc.

Reply:
Already on a card.